tests/e2e/sample-applications/virt-todo/test.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
```

Comment:
in general,
maybe add some curl examples for the requests.
in addition to the python

Reply:
ok added to readme, thanks